### ISSUE TITLE
Chargebee iFrame not loading during Cypress test (Uncaught (in promise) Error: No message target specified)

### ISSUE THREAD

**Issue by TedisAgolli** [chargebee/chargebee-js-wrappers#152]
Hello!

I am trying to run a Cypress test which loads a page and fills in the Chargebee information.
The Chargebee iFrame with the payment information fails to load with an error
`Uncaught (in promise) Error: No message target specified`
![image](https://github.com/chargebee/chargebee-js-wrappers/assets/10348514/8f7771a0-97fa-4c15-925a-722e5a96576f)

This is what I'm trying to load.
```jsx
       <CardComponent
          ref={cardRef}
          className='w-100'
        >
          <CardNumber placeholder='4111 1111 1111 1111' className='field empty' />
          <CardExpiry placeholder='MM / YY' className='field empty' />
          <CardCVV placeholder='CVV' className='field empty' />
        </CardComponent>
```

The page loads correctly in both dev and prod mode, when running outside of Cypress.
I can verify that `Chargebee.getInstance()` returns a valid instance, even inside the Cypress runner.
The iFrame is present on the page, but it's body is empty.

I tried chasing the error down in this [file](https://js.chargebee.com/assets/cbjs-2024.01.10-07.24/v2/master-53b7df5894b6cf02ffd7.js) but was not able to understand what it means. 

Any guidance is appreciated.

Thank you in advance!

**Comment by Brandons404** [comment]
Were you able to find a solution to this? The error message itself is less than helpful.

**Comment by TedisAgolli** [comment]
Nope. We've also opened up a support ticket and still haven't been able to figure it out.

**Comment by rainkinz** [comment]
Same issue. Very annoyingly useless message.

**Comment by cb-dinesh** [comment]
Hi, Sorry for the delay in addressing this. Let me look into it and get back.

**Comment by rainkinz** [comment]
Thanks, it's probably some setup issue, but very hard to figure out. I tried my setup with the example app and it works fine, with my app, which is a nextjs app (and I tried remix), it is not working. Do these wrappers actually work with React 18?

**Comment by rainkinz** [comment]
Any update?

**Comment by dbrad311** [comment]
@cb-dinesh Any update?

**Comment by chuttam** [comment]
Circling back here since I too recently encountered this exception in console with the Chargebee JS card component

 In my case the DOM structure surrounding the Chargebee JS component was incorrect, missing a `</div>` tag along the way. As a result, Chrome was unable to trigger the necessary events to mount the JS component into the DOM.

Incidentally, Firefox is more tolerant of DOM tree tag mismatches, and I did not see the uncaught promise rejection error in my specific case with that browser.. just with Chrome. This may help identify a root cause in your case too.

I understand this may not be the issue for more visitors here, but hopefully it helps someone here with similar root cause to my issue.
